packages/fgtclb/academic-partners/Resources/Private/Templates/Partner/Map.html

```diff
@@ -3,68 +3,36 @@
-    <f:section name="Main">
-        <f:asset.css identifier="partnerC1" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css" />
-        <f:asset.script identifier="partnerS1" src="https://unpkg.com/leaflet@1.9.4/dist/leaflet.js" />
-    
-        <f:asset.css identifier="partnerC3" href="https://unpkg.com/leaflet.markercluster@1.4.1/dist/MarkerCluster.Default.css" />
-        <f:asset.script identifier="partnerS3" src="https://unpkg.com/leaflet.markercluster@1.4.1/dist/leaflet.markercluster.js" />
-    
-        <f:asset.css identifier="partnerC2" href="EXT:academic_partners/Resources/Public/Css/map.css" />
-        <f:asset.script identifier="partnerS2" src="EXT:academic_partners/Resources/Public/JavaScript/map.js" />
-        
-        <f:if condition="{settings.hideSorting}==false || {settings.hideFilter}==false">
-            <div class="partner-filter">
-                <f:form
-                    action="list"
-                    id="partner-filter"
-                    name="demand"
-                    object="{demand}"
-                >
-                    <div class="row">
-                        <f:if condition="{settings.hideSorting}==false">
-                            <f:render
-                                partial="Partner/DemandSorting"
-                                arguments="{_all}"
-                            />
-                        </f:if>
-                        <f:if condition="{settings.hideFilter}==false">
-                            <f:render
-                                partial="Partner/DemandCategories"
-                                arguments="{_all}"
-                            />
-                        </f:if>
-                    </div>
-                </f:form>
-            </div>
-        </f:if>
-        <div id="map" class="academicpartners__map"></div>
-        <ul id="partners" class="partners">
-            <f:for each="{partners}" as="partner">
-                <li
-                    id="partner-{partner.uid}"
-                    class="partner"
-                    data-distance=""
-                    data-marker=""
-                    data-description="{partner.description}"
-                    data-name="{partner.title}"
-                    data-lat="{partner.geocodeLatitude}"
-                    data-lng="{partner.geocodeLongitude}"
-                >
-                    <f:link.page pageUid="{partner.uid}">
-                        {partner.title}
-                    </f:link.page> 
-                    ({partner.addressStreet} {partner.addressStreetNumber},
-                    {partner.addressZip} {partner.addressCity},
-                    <f:if  condition="{partner.addressCountryLocalizedNameLabel}">{f:translate(key: '{partner.addressCountryLocalizedNameLabel}')}</f:if>)
-                </li>
-            </f:for>
-        </ul>
-    </f:section>
+
+<f:asset.css identifier="partnerC2" href="EXT:academic_partners/Resources/Public/Css/map.css" />
```

Comment:
I guess it would be good to have that as a partial for css/js (single or two) which would make it easier in project to extend/replace them ....

@dot3media eventually for a general stragety / later point ?